config/assets/default.js

```diff
@@ -1,74 +1,4 @@
 module.exports = {
-  client: {
-    lib: {
-      // Load Angular-UI-Bootstrap module templates for these modules:
-      uibModuleTemplates: [
-        'datepicker',
-        'modal',
-        'popover',
-        'progressbar',
-        'tabs',
-        'tooltip',
-        'typeahead',
-      ],
-      js: [
-        // Non minified versions
-        'node_modules/jquery/dist/jquery.js',
-        'node_modules/angular/angular.js',
-        'node_modules/angular-aria/angular-aria.js',
-        'node_modules/angular-resource/angular-resource.js',
-        'node_modules/angular-animate/angular-animate.js',
-        'node_modules/angular-touch/angular-touch.js',
-        'node_modules/angular-sanitize/angular-sanitize.js',
-        'node_modules/angular-message-format/angular-message-format.js',
-        'node_modules/angulartics/src/angulartics.js',
-        'node_modules/angular-ui-router/release/angular-ui-router.js',
-        'node_modules/angular-ui-bootstrap/src/buttons/buttons.js',
-        'node_modules/angular-ui-bootstrap/src/collapse/collapse.js',
-        'node_modules/angular-ui-bootstrap/src/dateparser/dateparser.js',
-        'node_modules/angular-ui-bootstrap/src/datepicker/datepicker.js',
-        'node_modules/angular-ui-bootstrap/src/debounce/debounce.js',
-        'node_modules/angular-ui-bootstrap/src/dropdown/dropdown.js',
-        'node_modules/angular-ui-bootstrap/src/isClass/isClass.js',
-        'node_modules/angular-ui-bootstrap/src/modal/modal.js',
-        'node_modules/angular-ui-bootstrap/src/multiMap/multiMap.js',
-        'node_modules/angular-ui-bootstrap/src/popover/popover.js',
-        'node_modules/angular-ui-bootstrap/src/position/position.js',
-        'node_modules/angular-ui-bootstrap/src/progressbar/progressbar.js',
-        'node_modules/angular-ui-bootstrap/src/stackedMap/stackedMap.js',
-        'node_modules/angular-ui-bootstrap/src/tabs/tabs.js',
-        'node_modules/angular-ui-bootstrap/src/tooltip/tooltip.js',
-        'node_modules/angular-ui-bootstrap/src/typeahead/typeahead.js',
-        'node_modules/moment/moment.js',
-        'node_modules/angular-moment/angular-moment.js',
-        'node_modules/medium-editor/dist/js/medium-editor.js',
```

Comment:
How does this get loaded? (It works, anyways!)

Reply:
This is loaded in `config/webpack/webpack.shims.js` as a global thing. I should be able to switch that to an import.